README.md

```diff
@@ -1,172 +1,54 @@
+* As only `import` and `export` are transpiled, those wanting to support older browsers should also pre-transpile any ES6 code into ES5.
+* What was previously in the `System` object, for example `System.import`, is now in `System.loader` (applications can of course supply an alias to ease the transition).
+* The previous `normalize` and `locate` methods have now been combined into the `resolve` method.
+* The `Module` constructor has been moved to the `Reflect` object.
+* Although the WhatWG loader will eventually define a `site` object, which will probably include some kind of module name/URL mapping, this is not currently defined and so is not included in the current polyfill; this means there is no longer any `baseURL` or `paths` support. For compatibility with script modules, the default loader only supports relative URLs `/`, `./` and `../` in `import` statements (this is relative to the URL of the importing module).
+
+As before, [SystemJs](https://github.com/systemjs/systemjs/) provides add-on functionality supporting legacy module formats, `System.register`, and module name/URL mapping in the default `System.loader`.
 
 ## Contributing
+Changing to the new loader spec is quite a lot of work, so contributions of all kinds are welcome.
+
 In lieu of a formal styleguide, take care to maintain the existing coding style. Add unit tests for any new or changed functionality. Lint and test your code using [grunt](https://github.com/cowboy/grunt).
 
 _Also, please don't edit files in the "dist" subdirectory as they are generated via grunt. You'll find source code in the "lib" subdirectory!_
 
-## Testing
-
-- `npm run test:node` will use node to  to run the tests
-- `npm run test:browser` will run `npm run test:browser-babel` and `npm run test:browser-traceur`
-- `npm run test:browser-[transpiler]` use karma to run the tests with Traceur or Babel.
-- `npm run test:browser:perf` will use karma to run benchmarks
-
-`npm run test:browser-[transpiler]` supports options after a double dash (`--`) :
-
-- You can use the `--polyfill` option to test the code with polyfill.
-
-- You can use the `--coverage` option to test and extract coverage info.
-
-- You can use the `--native-iterator` option to run extra tests that check that `for ... of` and other iterator syntaxes work with es6-module-loader
-
-- You can use the `--force-map-polyfill` option to simulate an environment where `Map` is not defined
-
-- You can use the `--saucelabs` option to use karma and saucelabs to run the tests in various browsers.
-Note: you will need to export your username and key to launch it.
-
-  ```sh
-  export SAUCE_USERNAME={your user name} && export SAUCE_ACCESS_KEY={the access key that you see once logged in}
-  npm run test:browsers -- --saucelabs
-  ```
-
 ## Credit
-Copyright (c) 2015 Luke Hoban, Addy Osmani, Guy Bedford
+Copyright (c) 2016 Luke Hoban, Addy Osmani, Guy Bedford
```

Comment:
Let's perhaps update this to:

##Contributors

Peter Robins
Joel Denning
Guy Bedford
Addy Osmani
Luke Hoban